kolibri/plugins/setup_wizard/assets/src/vue/index.vue

```diff
@@ -8,14 +8,19 @@
         <h2 class="title">{{ $tr('deviceOwner') }}</h2>
         <div class="description">{{ $tr('deviceOwnerDescription') }}</div>
         <div class="creation-form">
-          <br><input :class="{ 'input-error': username_error }" type="text" v-model="username" :placeholder="$tr('username')" :aria-label="$tr('username')"><br>
-          <br><input :class="{ 'input-error': password_error }" type="password" v-model="password" :placeholder="$tr('password')" :aria-label="$tr('password')"><br>
-          <br><input :class="{ 'input-error': password_error }" type="password" v-model="confirm_password" :placeholder="$tr('confirmPassword')" :aria-label="$tr('confirmPassword')">
+          <h3>{{ $tr('username') }}:</h3>
```

Comment:
H2 above still, but H3 element here for labels does not make much sense...

If it's only for stylistic reasons, maybe change to `<label class="h3">`, and turn line 135 style  `h3` below to class `.h3`.